mlflow/models/utils.py

```diff
@@ -66,6 +66,13 @@
 EXAMPLE_DATA_KEY = "inputs"
 EXAMPLE_PARAMS_KEY = "params"
 EXAMPLE_FILENAME = "input_example.json"
+SERVING_INPUT_PATH = "serving_input_path"
+SERVING_INPUT_FILENAME = "serving_input.json"
+# re-define here to avoid circular import from
```

Comment:
Can we import within function scope to avoid duplicating definition?

Reply:
Unfortunately importing within function doesn't work because mlflow-skinny can use _Example but scoring_server can't be imported in mlflow-skinny. I'm planning to refactor those things to scoring_server/utils.py file in a follow-up PR and then we can import from utils file even in mlflow-skinny!